src/dispatch/messaging/email/utils.py

```diff
@@ -45,19 +46,26 @@ def get_template(message_type: MessageType):
         ),
     }
 
-    template_path, description = template_map.get(message_type, (None, None))
+    template_key, description = template_map.get(message_type, (None, None))
 
-    if not template_path:
+    if not template_key:
         raise Exception(f"Unable to determine template. MessageType: {message_type}")
 
-    return env.get_template(os.path.join("templates", template_path)), description
+    try:
+        template_path = os.path.join("templates", "project_id", f"{project_id}", template_key)
+        template = env.get_template(template_path)
+    except jinja2.exceptions.TemplateNotFound:
+        template_path = os.path.join("templates", template_key)
+        template = env.get_template(template_path)
```

Comment:
```suggestion
    try:
        template_path = os.path.join("templates", "project_id", f"{project_id}", template_key)
    except jinja2.exceptions.TemplateNotFound:
        template_path = os.path.join("templates", template_key)

template = env.get_template(template_path)
```

Reply:
we want to call "get template" to be able to catch TemplateNotFound :)